config/locales/client.en.yml

```diff
@@ -52,6 +52,9 @@ en:
       download_csv: "CSV"
       others_dirty: "A query has unsaved changes that will be lost if you navigate away."
       run_time: "Query completed in {{value}} ms."
+      single_result_count: "{{value}} result."
+      total_result_count: "{{value}} results."
+      result_limit_reached: "Showing top {{value}} results."
```

Comment:
This text is hard to translate, because translators don't know which number `{{value}}` will be. There are multiple ways to solve this:

1. Pluralize the string with `one` and `other`, even though the number will always stay the same.
2. Replace `{{value}}` with the actual number 250.
3. Add a comment in the line above the translation.

I'd suggest you go with 1. It might look ridiculous in English and many other languages, but it's the best way to allow translation in every language.

Reply:
> 1. Pluralize the string with one and other, even though the number will always stay the same.

That sounds great, I've done this now but is this YAML acceptable? Or should I find a way to make it work with just `one` and `other`?
```yml
result_count:
  one: "%{count} result."
  other: "%{count} results."
  max: "Showing top %{count} results."
```

I need to say `top 250 returned` so that it's clear that all results have not been displayed.
This has been discussed here: https://meta.discourse.org/t/have-data-explorer-show-number-of-results/96687/2?u=rishabhn